zstacklib/zstacklib/utils/iscsi_nl.py

```
@@ -0,0 +1,484 @@
+#!/usr/bin/env python2
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)